src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -261,6 +281,10 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
     this._clickAction = this._clickAction || this._options.clickAction;
   }
 
+  getAttrBlacklist() {
```

Comment:
Method should start with `_`, also please change `blacklist` to `blocklist` (this is a change we've been making across angular's codebase)